Validator - Spencer.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+from dateutil import parser
+# Gets todays date
+today = datetime.today()
+# Puts it into the dd/mm/yyyy format
+today = today.strftime("%d/%m/%Y")
+
+
+def DateValidation():
```

Comment:
You could maybe add the date validation as a parameter instead of an input, otherwise this part looks good!

Reply:
I have changed this in the recent commit